tests/tracking/fluent/test_fluent.py

```diff
@@ -2083,6 +2080,65 @@ def predict(model_input):
     assert new_model.model_id != model_id
 
 
+def test_set_active_model_in_databricks_serving():
+    with mock.patch(
+        "mlflow.tracking.fluent.is_in_databricks_model_serving_environment",
+        return_value=True,
+    ):
+        model = set_active_model(name="test_model")
+        assert mlflow.get_active_model_id() == model.model_id
+        assert _MLFLOW_ACTIVE_MODEL_ID.get() == model.model_id
+
+
+def test_get_active_model_id_global():
+    model = mlflow.create_external_model()
+
+    with ThreadPoolExecutor(max_workers=4) as executor:
+        futures = [executor.submit(set_active_model, model_id=model.model_id) for i in range(4)]
+        for f in futures:
+            f.result()
+
+    assert mlflow.get_active_model_id() is None
+    assert _get_active_model_id_global() == model.model_id
+
+    with ThreadPoolExecutor(max_workers=4) as executor:
+        futures = [executor.submit(set_active_model, name=f"test_model_{i}") for i in range(4)]
+        for f in futures:
+            f.result()
+
+    with mock.patch("mlflow.tracking.fluent._logger.warning") as mock_warning:
+        assert _get_active_model_id_global() is None
+        assert "Failed to get one active model id from all threads" in mock_warning.call_args[0][0]
```

Comment:
Let's use `any` and `call_args_list`. Any `warning` call would break this assert statement.